tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1824,107 +1797,85 @@ def test_continuous_writes_during_pod_restart(self):
-        LOGGER.info("Step 4: Verify services status on remaining pods %s are in online state",
-                    remain_pod_list)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=remain_pod_list, fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 4: Verified services on remaining pods %s are in online state",
-                    remain_pod_list)
-
-        LOGGER.info("Step 5: Perform READ/WRITEs/VERIFY with variable object sizes in background")
+        LOGGER.info("STEP 1: Perform WRITEs/READs/Verify with variable object sizes")
         users = self.mgnt_ops.create_account_users(nusers=1)
         self.test_prefix = 'test-34076'
         self.s3_clean.update(users)
-        args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
-                'skipcleanup': True, 'nclients': 1, 'nsamples': 30}
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipcleanup=True)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Performed WRITEs/READs/Verify with variable sizes objects.")
+        LOGGER.info("Step 2: Shutdown random data pod by deleting deployment and "
+                    "verify cluster & remaining pods status")
+        resp = self.ha_obj.delete_kpod_with_shutdown_methods(
+            master_node_obj=self.node_master_list[0], health_obj=self.hlth_master_list[0],
+            down_method=const.RESTORE_DEPLOYMENT_K8S)
+        # Assert if empty dictionary
+        assert_utils.assert_true(resp[1], "Failed to shutdown/delete pod")
+        pod_name = list(resp[1].keys())[0]
+        self.deployment_name = resp[1][pod_name]['deployment_name']
+        self.deployment_backup = resp[1][pod_name]['deployment_backup']
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown data pod %s. Verified cluster and "
+                    "services states are as expected & remaining pods status is online.", pod_name)
+        self.restore_pod = True
+        output = Queue()
+        event = threading.Event()  # Event to be used to send intimation of pod restart
+        LOGGER.info("Step 3: Start READ/WRITEs/VERIFY with variable object sizes in background")
+        log_prefix = self.test_prefix
+        if CMN_CFG["dtm0_disabled"]:
+            self.test_prefix_deg = 'test-34076-deg'
+            log_prefix = self.test_prefix_deg
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': log_prefix,
+                'skipcleanup': True, 'nclients': 1, 'nsamples': 30, 'setup_s3bench': False}
         thread = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,), kwargs=args)
```

Comment:
We need 2 different threads for READs and WRITEs here

Reply:
Did not change the original logic here, just added extra steps. So this won't work?

Reply:
Yes, I think we would need two different threads.

Reply:
This is not done @sonalk0209  ?

Reply:
yes was confirming first before making changes, will do now